README.md

```diff
@@ -24,6 +24,25 @@ The best way to install Kdyby/Console is using  [Composer](http://getcomposer.or
 $ composer require kdyby/console
 ```
 
+Then enable the extension in your `config.neon`:
```

Comment:
I don't want this in the readme, it adds work for me to maintain two documents :-/